app/js/main.js

```diff
@@ -749,112 +762,157 @@ function notifyError(msg) {
 }
 
 /**
- * Display top menu
+ * Display a custom confirm message
+ *
+ * @param {String|Nunber} [msg=""] The confirm message to display.
```

Comment:
Did you install the FuncDocr Brackets extension? It really helps with JSDoc notations.